src/fights.ts

```diff
@@ -1389,7 +1394,7 @@ const freeFightSources = [
       if (have($effect`Shadow Affinity`)) return true;
       if (get("_shadowAffinityToday")) return false;
 
-      if (rufusTarget() === null) return true;
+      if (ClosedCircuitPayphone.rufusTarget() === null) return true;
```

Comment:
I think this is more legible using `!ClosedCircuitPayphone.rufusTarget()` instead of checking against null, but that is a matter of opinion and is extremely not a blocker